README.rst

```diff
@@ -27,6 +27,7 @@ The preferred way is that you put a link to your own repository here:
 - https://github.com/Alex131089/bbbs (Bash Borg Backup System – wrappers to handle pull-mode operation. Also comes with installer and ssh key provisioning)
 - https://github.com/makeITyourway/borg_wrapper (Another Borg wrapper enables you to handle multiple projects and configs per project)
 - https://github.com/engelant/borg-cygwin (Cygwin based Windows installer creator, with VSS based backup script)
+- https://github.com/witten/borgmatic (Python wrapper - Works with config files)
```

Comment:
oh, that one was missing still?

yeah, it should definitely be on that list but can we have some better description?

"Python wrapper" (borg wrapper?) and "works with config files" doesn't say much about why it is actually useful.

Reply:
i did my best, no idea what to say since I only use borgmatic, I have no reference to compare with